src/test/java/com/suse/saltstack/netapi/parser/SaltStackParserTest.java

```diff
@@ -0,0 +1,44 @@
+package com.suse.saltstack.netapi.parser;
+
+import com.suse.saltstack.netapi.results.SaltStackJob;
+import com.suse.saltstack.netapi.results.SaltStackResult;
+import com.suse.saltstack.netapi.results.SaltStackToken;
+import com.suse.saltstack.netapi.utils.SaltStackClientUtils;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class SaltStackParserTest {
+
+
+    @Test
+    public void testSaltStackJobParser() throws Exception {
+        InputStream is = this.getClass().getResourceAsStream("/minions_response.json");
+        SaltStackResult<List<SaltStackJob>> result = SaltStackParser.JOB.parse(is);
+        assertNotNull("failed to parse", result);
+        assertEquals("unable to parse jid", "20150211105524392307", result.getResult().get(0).getJid());
+    }
+
+
+    @Test
+    public void testSaltStackStringParser() throws Exception {
+        InputStream is = this.getClass().getResourceAsStream("/logout_response.json");
+        SaltStackResult<String> result = SaltStackParser.STRING.parse(is);
+        assertNotNull(result);
+    }
+
+
```

Comment:
Please put only one newline between those methods and at the beginning of the class. Thanks!